### ISSUE TITLE
Details on pre-training MASS with other English-only data

### ISSUE THREAD

**Issue by leekum2018** [microsoft/MASS#60]
Hi, thanks for sharing this work!  Now I have a question about the details what if I want to pre-train MASS with other English-only data.  How to turn the raw data into Byte-Pair Encoding?  And can I use the same script as followed?  Thanks.
`python train.py                                      \`
`--exp_name mass_english                      \`
`--data_path ./data/processed/en/                     \`
`--lgs 'en'                                           \`
`--mass_steps 'en'                                    \`
`--encoder_only false                                 \`
`--emb_dim 1024                                       \`
`--n_layers 6                                         \`
`--n_heads 8                                          \`
`--dropout 0.1                                        \`
`--attention_dropout 0.1                              \`
`--gelu_activation true                               \`
`--tokens_per_batch 3000                              \`
`--optimizer adam_inverse_sqrt,beta1=0.9,beta2=0.98,lr=0.0001 \`
`--epoch_size 200000                                  \`
`--max_epoch 100                                      \`
`--eval_bleu true                                     \`
`--word_mass 0.5                                      \`
`--min_len 5                                          \`
`--english_only true`

**Comment by StillKeepTry** [comment]
It may need to transfer the raw data into bpe data by your self. Using FastBPE and mosesdecoder.
